Remotion/Core/Core/Collections/ImmutableSingleton.cs

```diff
@@ -40,10 +41,11 @@ public sealed class ImmutableSingleton<T> : IReadOnlyList<T>
   {

```

Comment:
ImmutableSingleton.cs(121, 16): [CS8603] Possible null reference return.
  ImmutableSingleton.cs(78, 22): [CS8603] Possible null reference return.

Reply:
IMO this file does not need a single annotation. The nullability of `T` dictates the nullability of parameters/return values in this file. If it is `T!`, the annotated signature would be `public static ImmutableSingleton<T> Create<T> ([NotNull] T item)`. It only was `[CanBeNull]` because of `T~`.

Please leave this discussion open for MK`s review.

Reply:
Yes. we should restore the CanBeNull, though, right?